araproc/analysis/standard_reco.py

```diff
@@ -3,11 +3,13 @@
 import math
 import os
 import ROOT
+from scipy.interpolate import Akima1DInterpolator
 import itertools
 
 from araproc.analysis import interferometry as interf
 from araproc.framework import constants as const
 from araproc.framework import map_utilities as mu
+from araproc.framework import waveform_utilities as wu
```

Comment:
I think `wfu` is pretty standard in araproc right now, so might be some argument for consistency.

Reply:
Done!